contents/spotlight/startup-documenso.md

```diff
@@ -0,0 +1,79 @@
+
+> **Putting it into practice:** Ask friends and family to star your repo, then widen the request to relevant communities you can reach out to. Closed source teams can find communities unique to their industry, and provoke conversations there. 
+
+### 2. Centralize the community 
+
+A hard-learned lesson for Documenso was that the team started off trying to build a following in Slack, which they came to realize wasn’t the best platform for their users. Eventually, they decided to migrate to Discord as a community hub.
+
+“Switching to Discord was a key moment for us,” says Florent. “We wanted to double down on our community building and make the whole project community driven — which meant we needed to move to a more developer-friendly environment.”
+
+As the community settled in, the team created topics for conversation and tried to encourage GitHub contributions and bug reports. This focus on GitHub was essential because, while Discord was a great place to engage users, GitHub was the _real_ heart of the project. 
+
+“We focus on Discord and Twitter now, but we want everything to link back to GitHub and compound the effects there. GitHub really is our main channel and our key focus. That’s where Documenso really lives. It doesn’t live on Documenso.com. It lives on GitHub.”
+
+> **Putting it into practice:** Do everything you can to direct users to GitHub, using CTAs and banners on other platforms. You can use issue templates and a comprehensive README to help users communicate with you and lower the barrier to entry. 
+
+### 3. Find reasons to celebrate
+
+Twitter and Discord certainly helped the team grow its GitHub presence, but it’s been the major announcements which have driven the biggest surges in activity. The biggest came when the team announced their pre-seed funding.
+
+“Announcing our raise brought a lot of attention and we funneled all of that to GitHub,” says Florent. “We suddenly had open roles coming up, validation of our idea, and a bounty program too — so a lot of people were encouraged to star the repo and follow what we do.”
+
+Inspired by these surges, Documenso now announces similar milestones whenever it can — posting about product updates and new contributions in order to continuously drive users back to the repo. Anything which helps excite and engage users, helps. 
+
+“We use Discord and Twitter to celebrate milestones, celebrate contributions, and contributions whenever we can,” says Florent. “We’re really thankful for every external contribution, no matter how big they are. Even if you just fix a typo in the README file, that's fantastic. We really want to highlight all those elements and celebrate them.”
+
+> **Putting it into practice:** Documenso encourages contributions with a bounty program that posts new issues into their Discord community. Swag is a powerful tool for incentivizing users too! 
```

Comment:
I hope people also contribute to the mission as we're trying to take on a $B-empire and build a beautiful, open document signing experience 😄